fw/http_parser.c

```diff
@@ -5926,68 +5917,78 @@ do {									\
 		__FSM_EXIT(CSTR_EQ);					\
 	}, curr_st, next_st)
 
-#define H2_TRY_STR_LAMBDA_BY_REF_finish(str, lambda1, lambda2, finish, state)\
+/**
+ * The very similar to @H2_TRY_STR_2LAMBDA(), but uses referrers for states.
+ */
+#define H2_TRY_STR_BY_REF(str, curr_st, next_st)			\
```

Comment:
Perhaps, only one macro definition should be left - H2_TRY_STR_2LAMBDA or H2_TRY_STR_BY_REF?

Reply:
Yeah, good point: `H2_TRY_STR_BY_REF` can be simply expressed via `H2_TRY_STR_2LAMBDA`